src/main/java/io/github/darealturtywurty/ancientology/core/materials/ArmorMaterials.java

```diff
@@ -0,0 +1,77 @@
+package io.github.darealturtywurty.ancientology.core.materials;
+
+import java.util.function.Supplier;
+
+import io.github.darealturtywurty.ancientology.Ancientology;
+import io.github.darealturtywurty.ancientology.core.init.ItemInit;
+import net.minecraft.sounds.SoundEvent;
+import net.minecraft.sounds.SoundEvents;
+import net.minecraft.world.entity.EquipmentSlot;
+import net.minecraft.world.item.ArmorMaterial;
+import net.minecraft.world.item.Items;
+import net.minecraft.world.item.crafting.Ingredient;
+
+public enum ArmorMaterials implements ArmorMaterial {
+   BONE(Ancientology.MODID + ":bone", 10, new int[]{1, 2, 3, 1}, 15, SoundEvents.ARMOR_EQUIP_LEATHER, 0.0F, 0.0F, () -> {
+      return Ingredient.of(Items.BONE);
+   }),
+   BRONZE(Ancientology.MODID + ":bronze", 12, new int[]{1, 4, 5, 2}, 9, SoundEvents.ARMOR_EQUIP_IRON, 0.0F, 0.0F, () -> {
+      return Ingredient.of(ItemInit.BRONZE_INGOT);
```

Comment:
Use the forge tag here.

Reply:
What is the forge tag?? No idea what that is.